docs/04_For_Developers/06_Project_Maintainer_Guide.md

```diff
@@ -0,0 +1,58 @@
+# Project Maintainer Guide
+RSS-Bridge project maintainers are responsible for:
+1. Tagging issues
+1. Reviewing and merging pull requests
+1. Creating releases
+
+Project maintainers should keep all communications related to the project open
+to the community (Preferably GitHub issues but IRC/Matrix is also allowed).
+
+## Issues
+Maintainers should triage and respond to issues if possible, making sure to
+specify the tag if possible. For example, the 'Bug Report' can be clarified to
+'Bridge-Broken' once that is established. Project maintainers are encouraged to
+mention the bridge maintainer to comment on issues related to a specific bridge.
+
+An issue should remain open until it is resolved by fixing the issue or adding
+the feature it describes. There are some exceptions:
+- In the case of a feature request that will not be added, the issue can be
+  closed with the 'wontfix' label after pinging other active maintainers.
+- If an issue is too obscure or unclear and the issue creator does not respond
+  to additional clarification questions for 1 month, these issues should be
+  closed as they cannot be acted upon.
+
+## Pull Requests
+Maintainers should thoroughly **review** and **test** pull requests before
+merging them. Most bridge authors use other bridges in the project for reference
+so it is important to maintain a high quality codebase so that it does not
+deteriorate.
+
+To merge a pull request, use the "Squash and Merge" option in GitHub to compress
+the commits to a single change. When doing this, remove the extra commit
+description. However, if multiple authors contribute to a pull request, do not
+remove the Co-authored-by lines at the end of the message.
+
+Maintainers must not push to master, but create pull requests instead.
+Maintainers should not merge their own pull requests except in the following
+cases:
+- The pull request received no reviews or comments for 2 weeks.
+- The pull request is a trivial change. This definition is a little flexible,
+  but the diff should not exceed 5 lines and typically targets only a single
+  bridge.
+
+## Releases
+Releases are created about every 6 months, though this schedule can be
```

Comment:
Due to the nature of the project it's kinda silly to have tagged releases. The two advantages I see is for third-party packagers and just to show that the project is alive and kicking. Other than that should probably tag a release when we intentionally break stuff for end users.

Reply:
Tagged releases will also be important when we allow using this as a library. Though I guess we can decide what to do about that when it happens since it will require some more changes – e.g. libraries are best served with semver so we would either need periodic releases (could be done with CI) or splitting bridges and library into a separate Composer project (e.g. like https://github.com/j0k3r/graby/tags and https://github.com/j0k3r/graby-site-config/tags).